src/use_case/my_folder/MyFolderInputData.java

```diff
@@ -0,0 +1,21 @@
+package use_case.my_folder;
+
+import entity.recipe.Recipe;
+
+import java.util.HashMap;
+
+public class MyFolderInputData {
+
+    final private String foldername;
+
+    public MyFolderInputData(String foldername) {
```

Comment:
Ok, just a clarification. MyFolder is something in our login view that we click, we get it to another view with a list of folders this user have. So why do you use foldername as an input.

Reply:
yes, i think it should input username